storage/2023-11-03/blob/blobs/get.go

```diff
@@ -65,14 +66,15 @@ func (c Client) Get(ctx context.Context, containerName, blobName string, input G
 	var resp *client.Response
 	resp, err = req.Execute(ctx)
 	if resp != nil {
-		result.Contents = &[]byte{}
 		result.HttpResponse = resp.Response
 
-		err = resp.Unmarshal(result.Contents)
+		respBody, err := io.ReadAll(resp.Body)
```

Comment:
it's possible for `resp.Body` to be nil here fwiw, so this is a potential crash-point:

```suggestion
		var respBody io.Reader
		if resp.Body != nil {
			respBody, err = io.ReadAll(resp.Body)
```